src/main/java/com/gooddata/http/client/JsonUtils.java

```diff
@@ -1,15 +1,16 @@
 /*
- * (C) 2021 GoodData Corporation.
+ * (C) 2022 GoodData Corporation.
  * This source code is licensed under the BSD-style license found in the
  * LICENSE.txt file in the root directory of this source tree.
  */
 package com.gooddata.http.client;
 
-import org.apache.commons.lang3.StringEscapeUtils;
+import org.apache.commons.text.StringEscapeUtils;
 
 /**
  * Internal JSON helper
  */
+@SuppressWarnings("java:S1118")
```

Comment:
yeah, in this case private constructor is better, since the class only has static method so we also might need to limit instantiation outside the class